src/Service/RequestProcessor.cpp

```diff
@@ -246,6 +252,8 @@ void RequestProcessor::processCommittedRequest(size_t count)
                 /// apply request
                 applyRequest(committed_request);
                 committed_queue.pop();
+                auto current_time = getCurrentTimeMilliseconds();
+                Metrics::getMetrics().UPDATE_LATENCY->add(current_time - committed_request.create_time);
```

Comment:
please update latency in `ConnectionHandler::updateStats`

Reply:
If we need to update the statistics in `ConnectionHandler::updateStats`, we need to keep the `request` in the `response`. I will open another pr to fix it.